util/text.py

```diff
@@ -166,3 +167,25 @@ def range_less_than(previous_state, current_input):
     vals_sparse = gather_nd(labels, indices, shape)
     
     return tf.SparseTensor(tf.to_int64(indices), vals_sparse, tf.to_int64(label_shape))
+
+# Validate and normalize transcriptions. Returns a cleaned version of the label
+# or None if it's invalid.
+def validate_label(label):
```

Comment:
I rebased my branch before merge and only this [line](https://github.com/mozilla/DeepSpeech/blob/d540a05c0911637277e81d4ddf1eb5c4a659e210/util/text.py#L181) ended being added to this function since there are multiple occurrences of curly braces in switchboard.